code/modules/mob/living/carbon/xenomorph/Facehuggers.dm

```diff
@@ -350,7 +350,7 @@
 	else
 		target.visible_message(SPAN_DANGER("[src] violates [target]'s face!"))
 
-	if(GLOB.round_statistics && ishuman(target))
+	if(GLOB.round_statistics && (ishuman(target)))
```

Comment:
```suggestion
	if(GLOB.round_statistics && ishuman(target))
```
this define has parenthesis around it anyway - is this necessary?

Reply:
good catch, it was a leftover from the yautja thing woops